### PR TITLE
Idva5-1168 check your answers update

### PR DESCRIPTION
**Pull request by ttingle-ch**
Updated check your answers to create the answers at the end of the journey
Removed adding answers to session throughout the journey
Answers are now translated into Welsh

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: locales/cy/what-is-your-role.json [hunk 1]
```diff
@@ -8,6 +8,13 @@
     "director": "Rwy'n gyfarwyddwr ar ",
     "generalPartner": "Rwy'n bartner cyffredinol o ",
 
+    "iAmADirector": "I am a director",
+    "iAmAmemberOfPartnership": "I am a member of the partnership",
+    "iAmAMember": "I am a member",
+    "iAmAMemberOfGoverningBody": "I am a member of the governing body",
+    "iAmEquivalentToDirector": "I am the equivalent to a director",
+    "IAmAGeneralPartner": "I am a general partner",
+
```
Comment (sardhani-ch) [inline locales/cy/what-is-your-role.json:17]: Can you add welsh at the end please

Comment (ttingle-ch) [reply locales/cy/what-is-your-role.json:17]: Done

**Review by sardhani-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by abhiahirv1 — APPROVED** [review]

**Pull request merged**